pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "flake8-pantsbuild"
+version = "0.1.0"
+description = "Various Flake8 lints used by the Pants project and its users."
+authors = ["Eric Arellano <ericarellano@me.com>"]
```

Comment:
Wasn't sure what to put here. Should we include the `pantsbuild@gmail.com` email? Credit the original authors of the `checker` wheel upon which this is based?

I wanted to leave this field off, but Poetry requires it. There's also an optional `maintainers` field.

Reply:
Perhaps follow pex's lead?

Reply:
https://github.com/pantsbuild/pex/blob/master/pyproject.toml#L1-L8

Reply:
Thanks for pointing to this! Updated.